resources/provider/migrations/timescale/20_v0.10.9.up.sql

```diff
@@ -0,0 +1,9 @@
+-- Resource: directconnect.gateways
+TRUNCATE aws_directconnect_gateways CASCADE;
```

Comment:
Instead can we use a DELETE statement like this:

```
DELETE FROM
    aws_directconnect_gateways a
        USING aws_directconnect_gateways b
WHERE
    a.cq_id < b.cq_id
    AND a.cq_fetch_date = b.cq_fetch_date
    AND a.account_id = b.account_id
    AND a.id = b.id
```

Reply:
Keeping data this way (when a PK has changed) doesn't work due to the way `cq_id` is calculated (using PKs) and how we remove those rows before inserting.